app/Filament/Admin/Resources/ProgramResource/RelationManagers/TeamsRelationManager.php

```diff
@@ -29,7 +42,26 @@ public function table(Table $table): Table
         return $table
             ->recordTitleAttribute('name')
             ->columns([
-                Tables\Columns\TextColumn::make('name'),
+                Tables\Columns\TextColumn::make('name')
+                    ->searchable()
+                    ->sortable(),
+                Tables\Columns\TextColumn::make('programs.name')
```

Comment:
Not sure we need the program name in the relation manager here, as we know what program we're looking at from the page.